### PR TITLE
German 3.22.0-PTU.9003376

### PR DESCRIPTION
**Pull request by ROBdk97**
@Auhrus can you check this please ^^
- Added/updated German Translations to match 3.22.0-PTU.9003376
- Additional fixes for some Liveries and Item names

### CODE REVIEW

File: data/Localization/german_(germany)/global.ini [hunk 1]
```diff
@@ -27877,7 +27888,6 @@ PU_GENOUTLAW2_M_CB_Relaxed_NoticeBulletImpactsClose_ThreatHigh_High_IG_002_ShitT
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeBulletImpactsClose_ThreatHigh_High_IG_003_SomeBastardsAfter=Irgendein Mistkerl ist hinter mir her!
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeCorpse_ThreatMed_IG_001_FoundAFrickin=Ich habe eine verdammte Leiche gefunden.
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeCorpse_ThreatMed_IG_002_WonderWhatThis=Ich frage mich, was er falsch gemacht hat.\n
-PU_Relaxed_NoticeCorpse_ThreatMed_IG_003_WhoKilledEm=Wer hat sie getötet? War das einer von euch?
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeCorpse_ThreatMed_IG_003_WhoKilledEm=Wer hat sie getötet? War das einer von euch?
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeFriendlyTakenDown_ThreatHigh_High_IG_001_SomeoneGotKnocked=Jemand wurde niedergeschlagen!
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeFriendlyTakenDown_ThreatHigh_High_IG_002_TakingInjuries=Verletzungen einstecken!
```
Comment (Brill65) [inline data/Localization/german_(germany)/global.ini:27893]: "Verletzungen Einstecken!" Klingt irgendwie falsch, vielleicht wäre "Bin verletzt!" oder "Wurde getroffen!" eine bessere übersetzung?

File: data/Localization/german_(germany)/global.ini [hunk 2]
```diff
@@ -61365,18 +61441,28 @@ mission_location_pyro_061e=der Wartungsbereich in PYAM-FARSTAT-2-3 am L3 Lagrang
 mission_location_pyro_061f=der Wartungsbereich in Checkmate am L4 Lagrange-Punkt von Pyro II
 mission_location_pyro_061f_02=Wartungsbereich-02 in Checkmate am L4 Lagrange-Punkt von Pyro II
 mission_location_pyro_061f_03=Wartungsbereich-03 in Checkmate am L4 Lagrange-Punkt von Pyro II
+mission_location_pyro_061g=Wartungsbereich-01 in Orbituary above Pyro III
```
Comment (Brill65) [inline data/Localization/german_(germany)/global.ini:61444]: Zeile 61444 "above" Pyro ändern in "über" Pyro

File: data/Localization/german_(germany)/global.ini (hunk 1 above)
Comment (ROBdk97) [reply data/Localization/german_(germany)/global.ini:27893]: Changed to "Mich hat´s erwischt!"

**Comment by ROBdk97** [comment]
@all-contributors please add @Brill65 for reviewing

**Comment by ROBdk97** [comment]
@all-contributors please add @Brill65 for review

**Comment by ROBdk97** [comment]
@all-contributors add @ROBdk97 for projectManagement

File: data/Localization/german_(germany)/global.ini [hunk 3]
```diff
@@ -43233,13 +43242,13 @@ crusader_hackprevent_desc_002=Das Datennetz bei ~mission(Location|Address) ist d
...
+crusader_platform_MultiEliminateSpecific_desc=Crusader Security hat Informationen erhalten, dass ein aktueller Überfall der Nine Tails, der bei ~mission(MissionCluster|Address) stattfindet, mehrere prominente Banden-Vollstrecker einschließt. Das Beenden dieses Überfalls und das Zur-Rechenschaft-Ziehen der Verantwortlichen ist für Crusader Security höchste Priorität. Deshalb suchen wir einen Auftragnehmer, der bereit ist, hinzugehen und die Anführer festzunehmen.\n\nAn jedem Angriffsort müssen Sie die Kräfte der Nine Tails dezimieren, bis Sie in der Lage sind, den zuständigen Vollstrecker herauszulocken.\n\nIn Anbetracht der Brutalität, die Nine Tails während ihrer jüngsten Belagerung eingesetzt hat, empfehlen wir den Auftragnehmern, sich auf den Einsatz von Gewalt vorzubereiten und dringend, Unterstützung mitzubringen.\n\nAUFTRAG GENEHMIGT DURCH:\nVerbindungsoffizier Gibbs\nID# 91G66BW0
```
Comment (Auhrus) [inline data/Localization/german_(germany)/global.ini:43251]: I think the mission variables are also wrong here, or why were they changed?
The same for line:
[43245](https://github.com/Dymerz/StarCitizen-Localization/pull/197/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R43245)
[43247](https://github.com/Dymerz/StarCitizen-Localization/pull/197/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R43247)
[43249](https://github.com/Dymerz/StarCitizen-Localization/pull/197/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R43249)

**Review by Auhrus — CHANGES_REQUESTED** [review]
checked

File: data/Localization/german_(germany)/global.ini [hunk 4]
```diff
@@ -44778,8 +44793,8 @@ ea_ui_scoring_ObjectiveDestroyed=Ziel zerstört
 ea_ui_scoring_PlayerBleeding=Blutungen
 ea_ui_scoring_RevCapture=Umkehrung der Erfassung
 ea_ui_scoring_SecondPlace=2. Platz
-ea_ui_scoring_SpecialEventPickup=Umschlag gesammelt
-ea_ui_scoring_SpecialEventPickupDenied=Umschlag verweigert
```
[the remark this answers is not in the document]
Comment (ROBdk97) [reply data/Localization/german_(germany)/global.ini:44782]: They changed it up and in the new English global.ini its now Coin

File: data/Localization/german_(germany)/global.ini (hunk 3 above)
Comment (ROBdk97) [reply data/Localization/german_(germany)/global.ini:43251]: They got changed in the English global.ini

**Review by Auhrus — APPROVED** [review]
Everything fits. Did not take the English global.ini into account.

**Pull request merged**